base/inc/fields/posts.class.php

```diff
@@ -52,6 +52,17 @@ public function __construct( $base_name, $element_id, $element_name, $field_opti
 				'description' => __( 'Taxonomies are groups such as categories, tags, posts and products.', 'so-widgets-bundle' ),
 			),
 
+			'tax_query_relation' => array(
+				'type' => 'radio',
+				'label' => __( 'Taxonomies Relationship', 'so-widgets-bundle' ),
+				'options' => array(
+					'OR' => __( 'OR', 'so-widgets-bundle' ),
+					'AND' => __( 'AND', 'so-widgets-bundle' ),
+				),
+				'description' => __( 'The relationship between the taxonomies. AND requires posts to have all of the specified taxonomies, while OR requires posts to have at least one of the specified taxonomies.', 'so-widgets-bundle' ),
```

Comment:
How about `The relationship between taxonomies. AND requires posts to have all of the specified taxonomies. OR requires posts to have at least one of the specified taxonomies.`